src/components/UserForm/BasicInformationSection.js

```diff
@@ -35,7 +35,8 @@ const BasicInformationSection = React.memo(
             ? userInterfaceLanguage
             : undefined
         const userDatabaseLanguageInitialValue = hasOption(
```

Comment:
Currently `hasOption` always returns `false` as its second argument is missing - meaning that `userDatabaseLanguageInitialValue` is always `undefined`.